_tour/default-parameter-values.md

```diff
@@ -62,3 +62,24 @@ public class Main {
 ```
 {% endtab %}
 {% endtabs %}
+
+### Default Parameters for Overloaded Methods
+
+It's important to notice that Scala doesn't allow having two methods with default parameters and with the same name (overloaded).
+
+For example, if you write `def f(a: Int = 1)`, the compiler generates `def f$default$1 = 1`. If you have two overloads with defaults on the same parameter position, then we would need a different naming scheme. In this situation, it is preferred to keep the generated byte-code stable over multiple compiler runs.
+
+The other reason is to avoid the ambiguity that can be caused due to the existence of default parameters. To illustrate the problem, let's consider the method declarations provided below:
```

Comment:
```suggestion
An important reason why is to avoid the ambiguity that can be caused due to the existence of default parameters. To illustrate the problem, let's consider the method declarations provided below:
```